src/screens/BeerOfTheDay/BeerOfTheDay.jsx

```diff
@@ -33,10 +43,22 @@ class BeerOfTheDayComponent extends PureComponent {
           onClick={this.getRandomBeer}
           text="Choose a beer"
         />
-        {beer && <RandomBeer randomBeer={beer} />}
+        {
+          name && <div key={id} className="random-beer">
```

Comment:
key is used for map purpose - no need to add it here

Reply:
Done!